experimental/dds/tree2/src/test/feature-libraries/editable-tree/mockData.ts

```diff
@@ -220,7 +222,16 @@ export const personData: ContextuallyTypedNodeDataObject = {
 
 export function personJsonableTree(): JsonableTree {
 	return jsonableTreeFromCursor(
-		cursorFromContextualData(fullSchemaData, rootPersonSchema.types, personData),
+		cursorFromContextualData(
+			{
+				schema: fullSchemaData,
+				getFieldGenerator: (key: FieldKey, schema: FieldStoredSchema): undefined => {
+					return;
+				},
+			},
```

Comment:
we have a lot of occurrences of this pattern. We might want to find a way to be more concise. We have a few options:

1. Just do this, but with more concise syntax. I see two ways to do this:
```suggestion
			{
				schema: fullSchemaData,
				getFieldGenerator(): undefined {
					return;
				},
			},
```
OR 

```suggestion
			{
				schema: fullSchemaData,
				getFieldGenerator: () => undefined,
			},
```

2. Provide a function to create these objects which provides some defaults.
3. Provide an easy way to get a reverent getFieldGenerator function (ex: get one from the a branch or the tree). This has the advantage of working when you want to do node identifier generation in addition to being pretty compact. (ex: tree.getFieldGenerator). You could also provide a constant noopGetFieldGenerator.

There are other options as well. I think moving to `getFieldGenerator: () => undefined` seems like a decent option for now.

Reply:
Why not just make `getFieldGenerator` optional, and use the `() => undefined` implementation as the default? That removes the need for these several places to have to pass anything in. Making properties optional for the sake of caller convenience is not always the right approach, but I think in this case there is a very clear and reasonable default (nothing).

Reply:
Looks like you already have defaultGetFieldGenerator (which is a version of option #3 I suggested). You could use that here.